README.md

```diff
@@ -283,7 +283,27 @@ However, timing attacks are real. And, the comparison function is _not_ time saf
 
 The characters that comprise the resultant hash are `./ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789$`.
 
-Resultant hashes will be 60 characters long.
+Resultant hashes will be 60 characters long and they will include the salt among other parameters, as follows:
+
+`$[algorithm]$[cost]$[salt][hash]`
+
+- 2 chars hash algorithm identifier prefix. `"$2a$" or "$2b$" or "$2y$"` indicates BCrypt
```

Comment:
OpenBSD based bcrypt does not generate $2y$ hashes.

Reply:
I took it from here: https://en.wikipedia.org/wiki/Bcrypt
 

> The prefix "$2a$" or "$2b$" (or "$2y$") in a hash string in a shadow password file indicates that hash string is a bcrypt hash in modular crypt format.[10] The rest of the hash string includes the cost parameter, a 128-bit salt (Radix-64 encoded as 22 characters), and 192 bits of the resulting hash value (Radix-64 encoded as 31 characters).[11] The Radix-64 encoding uses the unix/crypt alphabet, and is not 'standard' Base-64.[12][13] The cost parameter specifies a key expansion iteration count as a power of two, which is an input to the crypt algorithm.


And a stack-overflow answer in which I based the te change, that also said so. If this node implementation does not generate $2y$, then I would agree on removing it.

Reply:
$2y$ is specific to John The Ripper derived hash libraries. Ours is based on OpenBSD, so we do not generate it.

Reply:
Removed. Thanks!